docs/tutorial/lambda-aws.txt

```diff
@@ -0,0 +1,114 @@
+============================================
+Deploy to Serverless AWS Lambda with BrefPHP
```

Comment:
"Serverless" seems unnecessary, at least as far as the page title goes. You end up clarifying this in the first paragraph.

```suggestion
Deploy to AWS Lambda with BrefPHP
```

Note: you named the file "lambda-aws" but refer to "AWS Lambda" here. Since the page title is incorporated into the URL, should they be consistent?

Reply:
Also using `Bref` as your other comments.
```suggestion
Deploy to AWS Lambda with Bref
```